ignite-worker/src/main/java/org/opennms/poc/ignite/worker/ignite/service/WorkflowService.java

```diff
@@ -31,16 +33,19 @@ public class WorkflowService implements Service {
 
     public WorkflowService(Workflow workflow) {
         this.workflow = Objects.requireNonNull(workflow);
+        if (serviceContext != null) log.info("############GOT SERVICE CONTEXT");
+        if (ignite != null) log.info("##########GOT IGNITION");
     }
 
     @Override
     public void init() {
-        log.info("{} SERVICE INITIALIZED", workflow.getUuid());
+        igniteLogger.info("############{} SERVICE INITIALIZED", workflow.getUuid());
```

Comment:
Does ignite logger support `{}` syntax?  I had the impression that doesn't work.

Reply:
unknown, that was already in there AFAIK